application/common/dos.py

```diff
@@ -134,22 +146,14 @@ def get_matching_dos_services(odscode: str, org_type_id: str) -> List[DoSService
 
 def get_new_odscode_for_dos(odscode: str, org_type_id: str) -> str:
```

Comment:
This function is irrelevant and should be removed. As dentists will always be 7 characters long